cmd/ocsp-updater/main_test.go

```diff
@@ -773,81 +773,92 @@ func TestMissingLogs(t *testing.T) {
 	}
 
 	testCases := []struct {
-		Logs                []*ctLog
-		GivenIDs            []string
-		ExpectedMissingLogs []*ctLog
+		description         string
+		logs                []*ctLog
+		givenIDs            []string
+		expectedMissingLogs []*ctLog
 	}{
-		// With `nil` logs, no log IDs are ever missing
 		{
-			Logs:                nil,
-			GivenIDs:            []string{testLogAID, testLogBID},
-			ExpectedMissingLogs: []*ctLog{},
+			"With `nil` logs, no log IDs are ever missing",
+			nil,
```

Comment:
I figured you were noting that field names were handy when table values got into messy structs, but given that this one is still pretty tidy I'm starting to suspect you are objecting more to the use of unnamed fields in general -- is that accurate?

If so, I'm happy to go back through and put all the field names back into the table declarations. This could be a debate that was had long before I showed up, but for what it's worth, it looks like skipping field names to make tables more succinct is a pretty regular practice: The testing for [fmt](https://golang.org/src/fmt/fmt_test.go#L134) does it, as does [strconv](https://golang.org/src/strconv/strconv_test.go#L18).